### ISSUE TITLE
Low RayShape precission

### ISSUE THREAD

**Issue by kubecz3k** [godotengine/godot#22249]
**Godot version:**
197a3b292487b57f5de4ae49ea2d61f2a6da7c7a

**OS/device including version:**
Ubuntu 18.04, GTX1060

**Issue description:**
Accuracy of the RayShape in current main (3.1) is very low, most of objects in my game are floating above the ground or are pushed into the ground. To tell truth already in 3.0 accuracy of RayShape was significantly lower than in 2.1, but now the problem is much worse. 

The same scene in 3 scenarios:
_current main (bullet):_
![zrzut ekranu z 2018-09-19 14-58-37](https://user-images.githubusercontent.com/6129594/45754983-65a22a00-bc1d-11e8-8651-cab1de06e9d4.png)

_stable 3.0.6 (bullet):_
![zrzut ekranu z 2018-09-19 14-58-30](https://user-images.githubusercontent.com/6129594/45754988-689d1a80-bc1d-11e8-865b-1964ff044429.png)

_stable 3.0.6  (godot physics engine):_
![zrzut ekranu z 2018-09-19 15-01-14](https://user-images.githubusercontent.com/6129594/45754992-6b980b00-bc1d-11e8-8708-9374112a5ce5.png)

**Steps to reproduce:**
Short version:
1. Download sample project.
2. Run `ConcaveCollisionShape.tscn` scene

Long version:
1. Create new scene
2. Create static body and its representation (in my sample/reproduction I'm using ConcavePolygonShape imported from Blender)
3. Create RigidBody with representation in character mode (above the static body)
4. Attach Ray Collision shape to the RigidBody
5. Run the simulation and observe the distance between rigid body and static body

**Minimal reproduction project:**
[RayShapePrecission.zip](https://github.com/godotengine/godot/files/2397160/RayShapePrecission.zip)

**Comment by kubecz3k** [comment]
cc @AndreaCatania
